### DISCUSSION TITLE
Highlight selected tags on the tags page

### DISCUSSION THREAD

**Discussion by plplmax** [dailydotdev/daily#1342] [Feature Requests]
It will be nice if you highlight selected tags on the [tags](https://app.daily.dev/tags) page.

"Choose tags to follow" section in "Feed settings" does not show all tags, therefore, does not show all selected tags.
Now we should use search field above "Choose tags to follow" to check if some tag is selected.

**Comment by idoshamun** [comment]
Hi @plplmax,

That's a great feedback. Thank you :)

**Comment by leonardopugliese** [comment]
Hi @idoshamun! Some news on this feature?

**Reply by idoshamun** [reply]
Nope. We don't have plans at the moment to make it happen. We may consider it for open source contribution though. I'll check internally

**Reply by rebelchris** [reply]
It's open for contribution now:
https://github.com/dailydotdev/daily/issues/1607
